blueocean-dashboard/src/main/js/components/Activity.jsx

```diff
@@ -45,33 +40,25 @@ EmptyState.propTypes = {
     showRunButton: bool,
     onNavigation: func,
 };
-
+@observer
 export class Activity extends Component {
+
     componentWillMount() {
         if (this.context.config && this.context.params) {
-            const {
-                params: {
-                    pipeline,
-                    organization,
-                },
-                config = {},
-            } = this.context;
-
-            config.pipeline = pipeline;
-            config.organization = organization;
-            this.props.fetchRuns(config);
+            const organization = this.context.params.organization;
+            const pipeline = this.context.params.pipeline;
+            this.pager = this.context.activityService.activityPager(organization, pipeline);
```

Comment:
I wonder if a better approach here would be to take the route params and transform it into the REST HAL href that represents the resource we are paging. Rather than creating our own keys (which will collide if we are using `fullName` in place of `pipeline.name`)...